apps/pollbook/backend/src/peer_app.ts

```diff
@@ -26,9 +26,13 @@ function buildApi(context: PeerAppContext) {
 
     getEvents(input: { lastEventSyncedPerNode: Record<string, number> }): {
       events: PollbookEvent[];
+      configurationInformation: PollbookConfigurationInformation;
       hasMore: boolean;
     } {
-      return store.getNewEvents(input.lastEventSyncedPerNode);
+      return {
+        ...store.getNewEvents(input.lastEventSyncedPerNode),
+        configurationInformation: store.getPollbookConfigurationInformation(),
+      };
```

Comment:
Is it necessary for `getEvents` to return the configuration information and events in the same call? It would be a little clearer if the configuration information was returned in a separate call but I understand this setup if it avoiding race conditions w/ separate calls would be annoying

Reply:
I'm trying to reduce the number of calls happening over the network as this is being called constantly.